pkg/splunk/enterprise/finalizers.go

```diff
@@ -42,7 +42,7 @@ func DeleteSplunkPvc(cr splcommon.MetaObject, c splcommon.ControllerClient) erro
 		component = "license-master"
 	case "SearchHeadCluster":
 		component = "search-head"
-	case "IndexerCluster":
+	case "IndexerCluster", "ClusterMaster":
 		component = "indexer"
```

Comment:
Should we make the component as "cluster-master" here?

Reply:
I thought about that but then I thought not to play around with it since it is based on the labels and if I change it I need to change the labels everywhere and need to change a bit of logic on CM side too. Hence for simplicity kept it like this.

Reply:
After the monitoring console changes come in, the code flow for clusterMaster will be different than the indexer, I can see it can cause some confusion there. Maybe I'll have to change this in the future